src/Sulu/Bundle/AdminBundle/Serializer/Subscriber/DropdownToolbarActionSubscriber.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Serializer\Subscriber;
+
+use JMS\Serializer\EventDispatcher\Events;
+use JMS\Serializer\EventDispatcher\EventSubscriberInterface;
+use JMS\Serializer\EventDispatcher\ObjectEvent;
+use Sulu\Bundle\AdminBundle\Admin\Routing\DropdownToolbarAction;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class DropdownToolbarActionSubscriber implements EventSubscriberInterface
```

Comment:
I'm thinking if we should start mark this kind of services as `@internal` so IDE will tell the devs not using this class.

Reply:
I think then we should add the `@internal` annotation a lot more often, and I am questioning if now is the right time to decide something like this.